src/System.IO.FileSystem/src/System/IO/FromApp.Interop.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+#pragma warning disable BCL0015
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport("FileApiInterop.dll", SetLastError = true, CharSet = CharSet.Unicode, ExactSpelling = true)]
+        private static extern bool CopyFileFromApp(string lpExistingFileName, string lpNewFileName, bool bFailIfExists);
+
+        internal static int CopyFile(string src, string dst, bool failIfExists)
+        {
+            src = PathInternal.EnsureExtendedPrefixOverMaxPath(src);
+            dst = PathInternal.EnsureExtendedPrefixOverMaxPath(dst);
+            if (!CopyFileFromApp(src, dst, failIfExists))
+            {
+                return Marshal.GetLastWin32Error();
+            }
+            return Errors.ERROR_SUCCESS;
+        }
+
+        [DllImport("FileApiInterop.dll", SetLastError = true, CharSet = CharSet.Unicode, ExactSpelling = true)]
+        private static extern bool CreateDirectoryFromApp(string lpPathName, ref SECURITY_ATTRIBUTES lpSecurityAttributes);
+
+        internal static bool CreateDirectory(string path, ref SECURITY_ATTRIBUTES lpSecurityAttributes)
+        {
+            // We always want to add for CreateDirectory to get around the legacy 248 character limitation
```

Comment:
This one is different from the others- the rest are all *OverMaxPath() which is why this one is called out.